css/style.css

```diff
@@ -0,0 +1,16 @@
+@media (min-width: 1281px) {
+#navbar{
+    width: 20%;
+    background-color: gray;
+    height: 100%;
+    background-color: white;
+}
+#main-doc{
+    width: 80%;
+}
```

Comment:
Please use a more semantic way of writing CSS. Here I'm talking about indentation between your CSS selectors and braces 👌🏾 🤗